cli/tests/end_to_end/extension.rs

```diff
@@ -185,3 +185,21 @@ pub fn async_state() {
         .run()
         .success();
 }
+
+#[tokio::test]
+pub async fn rest_api() {
+    let test_cli = TestCli::builder().with_config(None).build();
+
+    test_cli
+        .extension(
+            "
+            const reply = await PhylumApi.fetchPhylum(ApiVersion.V0, '/health');
```

Comment:
Looks like this was missed.

```suggestion
            const reply = await PhylumApi.fetch(ApiVersion.V0, '/health');
```